### ISSUE TITLE
FEAT: Read more option for long messages

### ISSUE THREAD

**Issue by r3yc0n1c** [RocketChat/EmbeddedChat#719]
As an EmbeddedChat developer

**I need to**:
Implement a "Read more" option for long messages.

**So That**:
Users can view long messages in a more manageable way without overwhelming the chat interface, improving user experience and readability.

**Acceptance Criteria**

- [ ] The "Read more" button is visible when a message exceeds a certain character length (e.g., 150 characters).
- [ ] Clicking "Read more" expands the message to show its full content.
- [ ] The "Read more" button changes to "Read less" once the full message is revealed, allowing users to collapse it back.
- [ ] The expanded message maintains proper formatting and displays without causing layout issues.
- [ ] The feature works consistently across different devices and screen sizes.

**Comment by r3yc0n1c** [comment]
I'm working on this

**Comment by arorabhavik1001** [comment]
Hey! Is this issue still open? I can help with the same

**Comment by r3yc0n1c** [comment]
I'm working on this

**Comment by sushen123** [comment]
hey @r3yc0n1c are you still working on this? If you are busy can I work on this?

**Comment by r3yc0n1c** [comment]
yes, this one is not only related to Embedded Chat anymore, so we need further confirmation from the team to make any changes

**Comment by Spiral-Memory** [comment]
Closing due to inactivity.
